Review.java

```diff
@@ -247,4 +247,74 @@ public static String fakeReview(String fileName)
      return fake;
 }
 
+public static String fakeReview(String fileName)
+
+
+
+
+ {
+
+
+     String text = textToString(fileName);
+
+
+    
+
+
+     for(int i = 0; i < text.length(); i++)
+
+
+     {
+
+
+        if(text.charAt(i) == '*')
+
+

```

Comment:
@xaamaro should lesson the amount of spaces between the code to make it easier to read.